CONTRIBUTING.md

```diff
@@ -41,11 +41,11 @@ Otherwise, to add to the dictionary:
 
    Format: `typo,correction[,correction...]`
 
-2. Code-gen the dictionary
+2. Verify (and postprocess) the dictionary
```

Comment:
I think the original intention was to run both steps here and then in Step 3, run the command to verify everything is resolved.

There is the annoying aspect though of needing to run verify before codegen